core/src/main/java/org/apache/servicecomb/core/provider/producer/ProducerBootListener.java

```diff
@@ -44,9 +44,15 @@
 
 public class ProducerBootListener implements BootListener {
   private static final Logger LOGGER = LoggerFactory.getLogger(ProducerBootListener.class);
+  
+  private static final String pattern = "/%s/%s.yaml";
 
   @Override
   public void onAfterTransport(BootEvent event) {
+    boolean exportToFile = DynamicPropertyFactory.getInstance()
+            .getBooleanProperty(DefinitionConst.SWAGGER_EXPORT_ENABLED, false).get();
```

Comment:
I'd prefer to true.

Reply:
giving the choice to users may be a good idea

Reply:
I mean the default value be true is better.